src/PrinterLog.php

```diff
@@ -182,13 +182,17 @@ public function showMetrics(Printer $printer)
 
             foreach ($metrics as $key => $value) {
                 $label = $this->getLabelFor($key);
-                if ($label && $value > 0) {
+                if ($label && array_sum($metrics) > 0) {
```

Comment:
Also, could you add a small code comment to indicate that the  `array_sum` part is done to check if at least one value is greater than 0 for all the possible metrics at a given date ?

This will help future readers as this part of GLPI's code can be a bit tricky to understand.